changelog.md

```diff
@@ -1,6 +1,8 @@
 ### v4.5.7
 - Fix: catch failed read more request Errors
+- Fix: read more footer save button RTL layout
 - Update: footer wordmark
+- Update: improve read more save page button presentation abilities
```

Comment:
I updated the branch this is to be merged into to master and the changelog was updated based on merging your other PR and natalia's, so you may want to update the changelog.